suite-common/wallet-core/src/accounts/accountsActions.ts

```diff
@@ -29,6 +29,64 @@ const removeAccount = createAction(
     }),
 );
 
+const prepareCreateAccountActionPayload = (
```

Comment:
Please don't use prepare prefix as it evokes prepareAction  https://redux-toolkit.js.org/api/createAction#using-prepare-callbacks-to-customize-action-contents

Reply:
fixed f3f6acc07d

Reply:
nit IMO

Reply:
Yes. That's a problem and not worth discussion / consideration?

Reply:
I meant by my previous answer, that it is opinionated and the renaming does not change anything. That's all.  But it seems like you really insist, so here is a fixup: 7b18890311. But please, after many previous discussions once again, don't be so pushy on small things as this one.

Reply:
Fixed in https://github.com/trezor/trezor-suite/pull/9705/commits/7b18890311401c36a31b658d89e5fc2193dbeeb1